searchlib/src/tests/queryeval/multibitvectoriterator/multibitvectoriterator_test.cpp

```diff
@@ -551,6 +551,7 @@ Verifier::create(bool strict) const {
     SearchIterator::UP iter(_is_and ? AndSearch::create(bvs, strict) : OrSearch::create(bvs, strict));
     auto mbvit = MultiBitVectorIteratorBase::optimize(std::move(iter));
     EXPECT_TRUE((bvs.size() < 2) || (dynamic_cast<const MultiBitVectorIteratorBase *>(mbvit.get()) != nullptr));
+    EXPECT_EQUAL(strict, Trinary::True == mbvit->is_strict());
```

Comment:
this is also good to verify, but it was not the strictness I was talking about (and it should be implicitly verified by the conformance test as well).

I was talking about the fact that non-first children to AND are *not strict* in production, but *are* strict in this test (before they are optimized away).

Reply:
Yes, I am aware of that. But as you say. That does not matter here. I just added a sanity check here to make it clear what we were testing.